lib/matplotlib/figunits.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import six
+import warnings
+import numpy as np
+from matplotlib.layoutbox import get_renderer
+
+"""
+Simple utility functions to parse and translate common
+drawing units
+"""
+
+suffixes = ['in', 'cm', 'mm', 'px', 'pt', 'em']
```

Comment:
global constants should be private (or IN_CAPS but probably not here)

Reply:
Should this module even be here?  Its so trivial, and I could likely just make it a utility function in `layoutbox`

Reply:
tbh I'm not even convinced we should support multiple units there -- should just be inches for now for consistency with figsize.  If we want to add support for multiple units, that should be as a separate PR (and starting with figsize) (I would probably support such a feature).